lib/client/api/sharing.js

```diff
@@ -95,6 +151,24 @@ module.exports = function (jiffClient) {
    */
   jiffClient.open = openProtocol.jiff_open.bind(null, jiffClient);
 
+  /**
+   * Open a secret share to reconstruct secret.
```

Comment:
This looks good, too. Just specify in the Docstring that it only works correctly on shares generated by gmw_share or associated funcs (i.e. gmw_and, gmw_xor)